src/sagemaker/local/data.py

```diff
@@ -0,0 +1,249 @@
+# Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+from __future__ import absolute_import
+
+import os
+import sys
+import tempfile
+from six.moves.urllib.parse import urlparse
+
+from sagemaker.amazon.common import read_recordio
+from sagemaker.local.utils import download_folder
+from sagemaker.utils import get_config_value
```

Comment:
nit: I am not sure if we agreed in following the Python Google style guide if that is case: [Use imports for packages and modules only, not for individual classes or functions](https://github.com/google/styleguide/blob/gh-pages/pyguide.md#22-imports).